igcollect/chrony.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+"""igcollect - Chrony NTP metrics collection
+
+Copyright (c) 2024 InnoGames GmbH
+"""
+
+import csv
+import time
+from argparse import ArgumentParser
+from subprocess import check_output
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='chrony')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    now = str(int(time.time()))
+
+    metrics = {
+        'activity': parse_activity(),
+        'tracking': parse_tracking(),
+    }
+
+    for category, data in metrics.items():
+        for metric, value in data.items():
+            print(f'{args.prefix}.{category}.{metric} {value} {now}')
+
+
+def run_chronyc_command(command):
+    chrony_info_raw = check_output(
+        ['/usr/bin/chronyc', '-c', command],
```

Comment:
Will this check ever run on FreeBSD? Chrony is not from base system but from a package. The path is `/usr/local/bin/chronyc`.

Reply:
I forgot to account for it. I pushed a fix that checks the platform.